nonsoolmateClient/src/takeTest/components/coachMark/PrevPageMark.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components";
+import { LeftCircleIc } from "@assets/index";
+import { commonFlex } from "style/commonStyle";
+export default function PrevPageMark() {
+  return (
+    <PrevPageMarkContainer>
+      <LeftCircleIc />
```

Comment:
```suggestion
      <LeftCircleIcon />
      
      
      const LeftCircleIcon = styled(LeftCircleIc)`
      widht: ~~~
      height: ~~~~`
```

우리 컨벤션에서는 svg를 styled-component화 할 때 Icon으로 하기로 했어요!!!
그리고, 
styled-component로 묶어주는 이유는,
나중에 우리 반응형을 할 때 묶어주어야 svg도 덩달아서 커지거나 줄어들 수 있어요!